ansible/roles/fanout/templates/arista_7260_connect.j2

```diff
@@ -10,6 +10,7 @@ vlan {{ dev_vlans | list | join(',') }}
 {% set peer_port = root_conn[intf]['peerport'] %}
 {% if peer_dev in lab_devices and 'Fanout' not in lab_devices[peer_dev]['Type'] and not deploy_leaf %}
  interface {{ intf }}
+  switchport
```

Comment:
Do we also need to update other templates with `switchport` configuration too?
For example - deploy templates - https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/fanout/templates/arista_7260cx3_deploy.j2#L94

Reply:
Probably yes. But for root fanout, currently this is the only used template. The other templates are for leaf fanout deploy. I think we can address the leaf fanout templates in other PRs.